libs/node-sdk/project.json

```diff
@@ -21,6 +21,7 @@
             "executor": "nx:run-commands",
             "options": {
                 "commands": [
+                    "echo \"export const version = '$(jq -r '.version' package.json)';\" > src/version.ts",
```

Comment:
Isn't this going to cause issues when you publish as there will be non committed changes in git.

Reply:
I've run the command before we do the commit, so the version number update should get checked in.